### ISSUE TITLE
seems MS has required github

### ISSUE THREAD

**Issue by franklingu** [upend/IF_MS_BUYS_GITHUB_IMMA_OUT#8]
https://www.theverge.com/2018/6/3/17422752/microsoft-github-acquisition-rumors

**Comment by vassudanagunta** [comment]
Yes. The number of people staring this repo has accelerated since that update.

If GitHub does in fact lose its independence, what I'd like to do is turn this repo into a place where the community can discuss options, including alternatives to GitHub.

I'll also like to gather like-minded people to help me launch [upend.org](http://upend.org) with the goal of promoting an internet not dominated by corporate interests, but rather people's interests.

**Comment by MaxShalom** [comment]
Seems official.
[https://www.bloomberg.com/news/articles/2018-06-03/microsoft-is-said-to-have-agreed-to-acquire-coding-site-github](https://www.bloomberg.com/news/articles/2018-06-03/microsoft-is-said-to-have-agreed-to-acquire-coding-site-github)

**Comment by franklingu** [comment]
I think, not sure though, the share price of MSFT will go up this Monday.
